locales/tr.json

```
@@ -1,565 +1,565 @@
 {
 	"revision": {
-		"revision": "Revisions",
+		"revision": "Düzeltmeler",
 		"download": "İndir",
 		"revert": "Geri al"
 	},
 	"common": {
 		"or": "veya",
 		"cancel": "İptal",
 		"ok": "Tamam",
-		"got-it": "Got it",
+		"got-it": "Anladım",
 		"period": "."
 	},
 	"help": {
 		"title": "Yardım",
 		"contact": {
 			"title": "Kişiler",
-			"messenger": "Talk to us",
-			"github": "Hata bildir",
-			"email": "Bize email gönderin",
-			"donate": "Buy us coffee"
+			"messenger": "Bizimle konuşun",
+			"github": "Bir sorun bildirin",
+			"email": "Bize e-posta gönderin",
+			"donate": "Bize kahve alın"
 		},
 		"documents": "Belgeler",
 		"features": "Özellikler",
-		"yaml-meta": "YAML Özbilgisi",
-		"slide": "Slayt Örneği",
-		"book": "Book Example",
-		"cheatsheet": "Cheatsheet",
+		"yaml-meta": "YAML meta verisi",
+		"slide": "Slayt örneği",
+		"book": "Kitap örneği",
+		"cheatsheet": "Kopya kağıdı",
 		"example": "Örnek",
 		"syntax": {
 			"h1": "Başlık",
-			"ul": "Sırasız Liste",
-			"ol": "Sıralı Liste",
-			"todo": "Yapılacaklar Listesi",
-			"blockquote": "Alıntı Bloğu",
-			"b": "Kalın yazı",
-			"i": "İtalik yazı",
+			"ul": "Sırasız liste",
+			"ol": "Sıralı liste",
+			"todo": "Yapılacaklar listesi",
+			"blockquote": "Alıntı öbeği",
+			"b": "Kalın yazı tipi",
+			"i": "Eğik yazı tipi",
 			"strike": "Üstüçizili",
-			"ins": "Eklenmiş yazı",
-			"mark": "Seçili yazı",
-			"anchor": "Link",
-			"img": "Resim",
+			"ins": "Eklenen yazı",
+			"mark": "İşaretli yazı",
+			"anchor": "Bağlantı",
+			"img": "Görüntü",
 			"code": "Kod",
-			"embeded": "Dış veriler",
-			"alert": "Burası bir uyarı bölgesi."
+			"embeded": "Dış veri",
+			"alert": "Bu bir uyarı alanıdır."
 		}
 	},
 	"signup": {
-		"signup": "Sign up",
-		"new": "New to HackMD? ",
+		"signup": "Kaydol",
+		"new": "HackMD'de yeni misiniz? ",
 		"eeSite": {
 			"terms": {
-				"1": "By clicking below, you agree to our ",
-				"2": "terms of service."
+				"1": "Aşağıya tıklayarak, hizmet şartlarımızı ",
+				"2": "kabul etmiş olursunuz."
 			}
 		}
 	},
 	"signin": {
-		"signin": "Sign in",
-		"signin-ldap" : "Sign in to {{name}} (LDAP)",
-		"signin-via-email": "Sign in via Email",
-		"choose": "Metot seçin",
-		"via": "{{text}} ile giriş yapın",
-		"email-or-username": "Email or Username",
-		"email-or-username-placeholder": "Your email or Username",
-		"email": "Email",
-		"email-placeholder": "Your email",
-		"password": "Password",
-		"password-placeholder": "Your password"
+		"signin": "Oturum aç",
+		"signin-ldap" : "{{name}} (LDAP) oturumu aç",
+		"signin-via-email": "E-posta aracılığıyla giriş yapın",
+		"choose": "Yöntem seçin",
+		"via": "%s ile giriş yapın"
```

Please keep the template string

```suggestion
		"via": "{{text}} ile giriş yapın"
```